mapflow/mapflow.py

```diff
@@ -737,7 +737,7 @@ def edit_provider_callback(self) -> None:
                 return
             else:
                 self.user_providers.append(new_provider)
-                provider_index = len(self.providers) - 1
+                provider_index = len(self.providers)
```

Comment:
Why was it -1? It selects provider above newly added then

Reply:
Hm, I don't know, but then wouldn't it select previous row when updating. But it selects the correct one 🤔